tests/models/soil/test_carbon.py

```diff
@@ -311,12 +344,30 @@ def test_calculate_labile_carbon_leaching(dummy_carbon_data, moist_temp_scalars)
     assert np.allclose(actual_leaching, expected_leaching)
 
 
-def test_calculate_direct_litter_input_to_lmwc():
+def test_calculate_pom_decomposition(dummy_carbon_data, moist_temp_scalars):
+    """Check that particulate organic matter decomposition is calculated correctly."""
+    from virtual_rainforest.models.soil.carbon import calculate_pom_decomposition
+
+    expected_decomp = [0.0038056940, 0.0104286084, 0.0092200655, 0.0014665616]
+
+    actual_decomp = calculate_pom_decomposition(
+        dummy_carbon_data["soil_c_pool_pom"],
+        dummy_carbon_data["soil_c_pool_microbe"],
+        moist_temp_scalars,
+    )
+
+    assert np.allclose(actual_decomp, expected_decomp)
+
+
+def test_calculate_direct_litter_input_to_pools():
     """Check direct litter input to lmwc is calculated correctly."""
     from virtual_rainforest.models.soil.carbon import (
-        calculate_direct_litter_input_to_lmwc,
+        calculate_direct_litter_input_to_pools,
     )
+    from virtual_rainforest.models.soil.constants import LITTER_INPUT_RATE
 
-    actual_input = calculate_direct_litter_input_to_lmwc()
+    actual_input_lmwc, actual_input_pom = calculate_direct_litter_input_to_pools()
 
-    assert np.isclose(actual_input, 0.00015697011)
+    assert np.isclose(actual_input_lmwc, 0.00015697011)
+    assert np.isclose(actual_input_pom, 0.00031394022)
+    assert np.isclose(actual_input_lmwc + actual_input_pom, LITTER_INPUT_RATE)
```

Comment:
Have you tested any of these functions with DataArrays? There might be some changes to the code required, like `.any()` or `.all()`

Reply:
I think basically all of these functions return either `numpy` arrays or floats (as discussed the functions returning floats will go soon). It's only the highest level functions in the `soil` module that return `DataArray`s